tests/TestJsonFileHandler.py

```diff
@@ -1,8 +1,9 @@
 import json
+import os
 from drops.helpers.JsonFileHandler import JsonFileHandler
 
 
-supported_json = "drops/supported.json"
+supported_json = "tests/test_supported.json"
```

Comment:
why this filepath?

Reply:
I changed the file path to a copy of supported.json to make sure I did not overwrite the original file.  This can be switched back to use the original supported.json.